_documentation/user/flags/forward-only.md

```diff
@@ -5,4 +5,19 @@ type: Bad
 summary: Can't drive in reverse.
 ---
 
-...
+The Forward Only flag prevents backwards movement, as if a tank has Oscillation-Overthruster and is within a building.
```

Comment:
Add link to OO flag and the OO flag name doesn't have a hyphen

Reply:
I feel the reference to the OO flag doesn't really add anything can can be removed.  This is a pretty straight forward bad flag so it's just going to have a very minimal description.

Reply:
Makes sense, was a bit concerned about having the flag document being too short.